sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
+openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
+openssl x509 -noout -text -in device_ec_cert.pem
+
+if (Test-Path device_cert_store.pem)
+{
+  Remove-Item device_cert_store.pem
+}
+
+Get-Content device_ec_cert.pem, device_ec_key.pem | Set-Content device_cert_store.pem
+
+Write-Output "IMPORTANT:"
+Write-Output "It is NOT recommended to use OpenSSL on Windows or OSX. Recommended TLS stacks are:"
+Write-Output "Microsoft Windows SChannel: https://docs.microsoft.com/en-us/windows/win32/com/schannel"
+Write-Output "OR"
+Write-Output "Apple Secure Transport : https://developer.apple.com/documentation/security/secure_transport"
+Write-Output "If using OpenSSL, it is recommended to use the OpenSSL Trusted CA store configured on your system."
+
+Write-Output "`nSAMPLE CERTIFICATE GENERATED:"
+Write-Output "Use the following command to set the environment variable for the samples:"
+if ($IsWindows) {
+  Write-Output "`n`tset AZ_IOT_DEVICE_X509_CERT_PEM_FILE=$(Resolve-Path device_cert_store.pem)"
+}
+else {
+  Write-Output "`n`texport AZ_IOT_DEVICE_X509_CERT_PEM_FILE=$(Resolve-Path device_cert_store.pem)"
+}
+
+Write-Output "`nDPS SAMPLE:"
+Write-Output "Upload device_ec_cert.pem when enrolling your device with the Device Provisioning Service."
```

Comment:
Are there instructions (that DPS team owns, NOT use) that are somewhat permalinked where we can explain where and how to upload?

Reply:
There are instructions in the Readme: https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/README.md#iot-hub-dps-certificate-sample

And in that subsection is a link to further instruction: https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-x509#create-a-device-enrollment-entry-in-the-portal

I don't want to put too much of that instruction in the generation file, to limit all the areas we need to update info if something was to change.